packages/core/src/runtime/start.ts

```diff
@@ -59,7 +59,7 @@ export async function start<TArgs extends unknown[], TResult>(
 ) {
   return await waitedUntil(() => {
     // @ts-expect-error this field is added by our client transform
-    const workflowName = workflow.workflowId;
+    const workflowName = workflow?.workflowId;
```

Comment:
nit: should this actually use the util to extract the worklfow name instead of ID?

Reply:
We want the ID actually (for the queue topic)